ERROR - 2024-06-13 01:28:38 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('p:localhost', 'root', Object(SensitiveParameterValue), 'ldr_db', 3306, '', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(true)
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\appldr\app\Controllers\Profil.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Profil->index()
#8 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profil))
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(true)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xamppp\htdocs\appldr\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\appldr\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\appldr\app\Controllers\Profil.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Profil->index()
#7 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profil))
#8 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#13 {main}
CRITICAL - 2024-06-13 01:28:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: dataProfil]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `profile`', [], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Profil.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Profil->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profil))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
ERROR - 2024-06-13 01:28:43 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('p:localhost', 'root', Object(SensitiveParameterValue), 'ldr_db', 3306, '', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(true)
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\appldr\app\Controllers\Profil.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Profil->index()
#8 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profil))
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(true)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xamppp\htdocs\appldr\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\appldr\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\appldr\app\Controllers\Profil.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Profil->index()
#7 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profil))
#8 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#13 {main}
CRITICAL - 2024-06-13 01:28:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: dataProfil]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `profile`', [], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Profil.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Profil->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profil))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 01:28:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-13 02:45:07 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('p:localhost', 'root', Object(SensitiveParameterValue), 'ldr_db', 3306, '', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(true)
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->testimoni()
#8 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(true)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xamppp\htdocs\appldr\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\appldr\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->testimoni()
#7 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#8 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#13 {main}
CRITICAL - 2024-06-13 02:45:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: testimoni]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `testimoni`', [], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Testimoni.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->testimoni()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 02:45:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 02:45:17 --> ErrorException: Undefined variable $title
[Method: GET, Route: testimoni]
in APPPATH\Views\testimoni\testimoni.php on line 23.
 1 APPPATH\Views\testimoni\testimoni.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $title', 'C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\testimoni.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\testimoni.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('testimoni/testimoni', [], true)
 5 APPPATH\Controllers\Testimoni.php(14): view('testimoni/testimoni', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->testimoni()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 02:45:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 02:45:19 --> ErrorException: Undefined variable $title
[Method: GET, Route: testimoni]
in APPPATH\Views\testimoni\testimoni.php on line 23.
 1 APPPATH\Views\testimoni\testimoni.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $title', 'C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\testimoni.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\testimoni.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('testimoni/testimoni', [], true)
 5 APPPATH\Controllers\Testimoni.php(14): view('testimoni/testimoni', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->testimoni()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 02:49:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 02:49:37 --> ErrorException: Undefined variable $title
[Method: GET, Route: testimoni]
in APPPATH\Views\testimoni\testimoni.php on line 23.
 1 APPPATH\Views\testimoni\testimoni.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $title', 'C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\testimoni.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\testimoni.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('testimoni/testimoni', [], true)
 5 APPPATH\Controllers\Testimoni.php(24): view('testimoni/testimoni')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->testimoni()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 02:51:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 02:51:43 --> ErrorException: Undefined variable $title
[Method: GET, Route: testimoni]
in APPPATH\Views\testimoni\testimoni.php on line 23.
 1 APPPATH\Views\testimoni\testimoni.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $title', 'C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\testimoni.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\testimoni.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('testimoni/testimoni', [], true)
 5 APPPATH\Controllers\Testimoni.php(24): view('testimoni/testimoni')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 02:58:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 03:00:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 03:03:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 03:12:48 --> ErrorException: Undefined variable $penggunaList
[Method: GET, Route: createTestimoni]
in APPPATH\Views\testimoni\create_testimoni.php on line 21.
 1 APPPATH\Views\testimoni\create_testimoni.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $penggunaList', 'C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\create_testimoni.php', 21)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xamppp\\htdocs\\appldr\\app\\Views\\testimoni\\create_testimoni.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('testimoni/create_testimoni', [], true)
 5 APPPATH\Controllers\Testimoni.php(29): view('testimoni/create_testimoni')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->formcreate()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 03:14:11 --> BadMethodCallException: Call to undefined method App\Models\M_testimoni::getProdukList
[Method: GET, Route: createTestimoni]
in SYSTEMPATH\Model.php on line 921.
 1 APPPATH\Controllers\Testimoni.php(29): CodeIgniter\Model->__call('getProdukList', [])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->formcreate()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 07:21:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 07:21:45 --> BadMethodCallException: Call to undefined method App\Models\M_testimoni::getProdukList
[Method: GET, Route: createTestimoni]
in SYSTEMPATH\Model.php on line 921.
 1 APPPATH\Controllers\Testimoni.php(29): CodeIgniter\Model->__call('getProdukList', [])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->formcreate()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:24:49 --> ParseError: Unmatched '}'
[Method: GET, Route: createTestimoni]
in APPPATH\Models\M_testimoni.php on line 40.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xamppp\\htdocs\\appldr\\app\\Models\\M_testimoni.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\M_testimoni')
 3 APPPATH\Controllers\Testimoni.php(15): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\M_testimoni')
 4 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Testimoni->__construct()
 5 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 07:25:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 07:27:57 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "pesanan/create_testimoni.php"
[Method: GET, Route: createTestimoni]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('pesanan/create_testimoni.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('pesanan/create_testimoni', [], true)
 3 APPPATH\Controllers\Testimoni.php(38): view('pesanan/create_testimoni', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->formcreate()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
ERROR - 2024-06-13 07:32:32 --> mysqli_sql_exception: Column 'gambar' cannot be null in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `te...', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xamppp\htdocs\appldr\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xamppp\htdocs\appldr\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(50): CodeIgniter\BaseModel->save(Array)
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->store()
#10 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#11 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#16 {main}
CRITICAL - 2024-06-13 07:32:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'gambar' cannot be null
[Method: POST, Route: storeTesti]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`gambar`, `uraian`) VALUES (:gambar:, :uraian:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Testimoni.php(50): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:32:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'gambar' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`gambar`, `uraian`) VALUES (NULL, \'uraian doang\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`gambar`, `uraian`) VALUES (NULL, \'uraian doang\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`gambar`, `uraian`) VALUES (:gambar:, :uraian:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Testimoni.php(50): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:32:32 --> [Caused by] mysqli_sql_exception: Column 'gambar' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `testimoni` (`gambar`, `uraian`) VALUES (NULL, \'uraian doang\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`gambar`, `uraian`) VALUES (NULL, \'uraian doang\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`gambar`, `uraian`) VALUES (NULL, \'uraian doang\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`gambar`, `uraian`) VALUES (:gambar:, :uraian:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Testimoni.php(50): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
ERROR - 2024-06-13 07:37:55 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `te...', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xamppp\htdocs\appldr\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xamppp\htdocs\appldr\app\Models\M_testimoni.php(16): CodeIgniter\Model->insert(Array)
#8 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(82): App\Models\M_testimoni->tambah(Array)
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->store()
#10 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#11 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#16 {main}
CRITICAL - 2024-06-13 07:37:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: POST, Route: storeTesti]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\Testimoni.php(82): App\Models\M_testimoni->tambah([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:37:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264275_e33b3cc29cbd21768bf3.png\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264275_e33b3cc29cbd21768bf3.png\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\Testimoni.php(82): App\Models\M_testimoni->tambah([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:37:55 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264275_e33b3cc29cbd21768bf3.png\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264275_e33b3cc29cbd21768bf3.png\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264275_e33b3cc29cbd21768bf3.png\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\Testimoni.php(82): App\Models\M_testimoni->tambah([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:39:50 --> Error: Class "App\Controllers\ProdukModel" not found
[Method: POST, Route: storeTesti]
in APPPATH\Controllers\Testimoni.php on line 17.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Testimoni->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
ERROR - 2024-06-13 07:40:40 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `te...', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xamppp\htdocs\appldr\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xamppp\htdocs\appldr\app\Models\M_testimoni.php(16): CodeIgniter\Model->insert(Array)
#8 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah(Array)
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->store()
#10 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#11 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#16 {main}
CRITICAL - 2024-06-13 07:40:40 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: POST, Route: storeTesti]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:40:40 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264440_aef75966f1bdbccb6736.png\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264440_aef75966f1bdbccb6736.png\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:40:40 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264440_aef75966f1bdbccb6736.png\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264440_aef75966f1bdbccb6736.png\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264440_aef75966f1bdbccb6736.png\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
ERROR - 2024-06-13 07:46:33 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `te...', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xamppp\htdocs\appldr\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xamppp\htdocs\appldr\app\Models\M_testimoni.php(16): CodeIgniter\Model->insert(Array)
#8 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah(Array)
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->store()
#10 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#11 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#16 {main}
CRITICAL - 2024-06-13 07:46:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: POST, Route: storeTesti]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:46:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264793_369c70e0d18e3895827d.png\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264793_369c70e0d18e3895827d.png\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:46:33 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264793_369c70e0d18e3895827d.png\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264793_369c70e0d18e3895827d.png\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264793_369c70e0d18e3895827d.png\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\M_testimoni.php(16): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
ERROR - 2024-06-13 07:48:05 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `te...', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `te...')
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `te...')
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `te...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xamppp\htdocs\appldr\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xamppp\htdocs\appldr\app\Models\M_testimoni.php(23): CodeIgniter\Model->insert(Array)
#8 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah(Array)
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->store()
#10 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#11 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#16 {main}
CRITICAL - 2024-06-13 07:48:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: POST, Route: storeTesti]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\M_testimoni.php(23): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:48:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264885_f4fa7961fa8ab3be94dc.png\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264885_f4fa7961fa8ab3be94dc.png\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\M_testimoni.php(23): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 07:48:05 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`ldr_db`.`testimoni`, CONSTRAINT `testimoni_ibfk_1` FOREIGN KEY (`id_pengguna`) REFERENCES `pengguna` (`id_pengguna`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264885_f4fa7961fa8ab3be94dc.png\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264885_f4fa7961fa8ab3be94dc.png\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (\'uraian doang\', \'1718264885_f4fa7961fa8ab3be94dc.png\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `testimoni` (`uraian`, `gambar`) VALUES (:uraian:, :gambar:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\M_testimoni.php(23): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\Testimoni.php(87): App\Models\M_testimoni->tambah([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 07:53:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 07:53:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 07:53:44 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: testimoni]
in APPPATH\Views\testimoni\testimoni.php on line 44.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('testimoni/testimoni', [], true)
 4 APPPATH\Controllers\Testimoni.php(29): view('testimoni/testimoni', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 08:01:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 08:01:47 --> Error: Cannot use object of type stdClass as array
[Method: GET, Route: testimoni]
in APPPATH\Views\testimoni\testimoni.php on line 44.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('testimoni/testimoni', [], true)
 4 APPPATH\Controllers\Testimoni.php(29): view('testimoni/testimoni', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 08:20:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 08:20:19 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "edit_testimoni.php"
[Method: GET, Route: testimoni/edit/5]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('edit_testimoni.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('edit_testimoni', [], true)
 3 APPPATH\Controllers\Testimoni.php(99): view('edit_testimoni', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->edit('5')
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
CRITICAL - 2024-06-13 11:14:58 --> ErrorException: Undefined variable $profile
[Method: GET, Route: profil]
in APPPATH\Views\freeuser\home\profil.php on line 17.
 1 APPPATH\Views\freeuser\home\profil.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $profile', 'C:\\xamppp\\htdocs\\appldr\\app\\Views\\freeuser\\home\\profil.php', 17)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xamppp\\htdocs\\appldr\\app\\Views\\freeuser\\home\\profil.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('freeuser/home/profil', [], true)
 5 APPPATH\Controllers\Home.php(16): view('freeuser/home/profil')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->profil()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
ERROR - 2024-06-13 11:15:23 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('p:localhost', 'root', Object(SensitiveParameterValue), 'ldr_db', 3306, '', 0)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(true)
#2 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `testimo...', Array, false)
#4 C:\xamppp\htdocs\appldr\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\appldr\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(29): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->index()
#8 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#9 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xamppp\htdocs\appldr\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(true)
#1 C:\xamppp\htdocs\appldr\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\appldr\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `testimo...', Array, false)
#3 C:\xamppp\htdocs\appldr\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\appldr\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\appldr\app\Controllers\Testimoni.php(29): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(933): App\Controllers\Testimoni->index()
#7 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
#8 C:\xamppp\htdocs\appldr\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xamppp\htdocs\appldr\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xamppp\htdocs\appldr\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xamppp\htdocs\appldr\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xamppp\htdocs\appldr\system\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdoc...')
#13 {main}
CRITICAL - 2024-06-13 11:15:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: testimoni]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `testimoni`.*, `pengguna`.`nama` as `nama_pengguna`
FROM `testimoni`
JOIN `pengguna` ON `pengguna`.`id_pengguna` = `testimoni`.`id_pengguna`', [], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Testimoni.php(29): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 11:15:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 11:16:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-06-13 11:16:14 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "testimoni/edit.php"
[Method: GET, Route: testimoni/edit/5]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('testimoni/edit.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('testimoni/edit', [], true)
 3 APPPATH\Controllers\Testimoni.php(99): view('testimoni/edit', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Testimoni->edit('5')
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Testimoni))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xamppp\\htdocs\\appldr\\public\\index.php')
INFO - 2024-06-13 11:19:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 11:21:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 11:30:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 11:32:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 12:24:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-13 12:25:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
